site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+
+In order for us to integrate the TypeScript typings, we need to adjust two sections of our `tsconfig.json`.
+
+{% instruction 'Add `scripthost` to `lib` section of `tsconfig.json`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/tsconfig.json' lines:6-14 %}
+
+Some of the Dojo 1 typings reference typings from this library.
+
+{% instruction 'Add the Dijit module typings to `include` section of `tsconfig.json`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/tsconfig.json' lines:27-31 %}
+
+By adding the modules to the scope of the TypeScript compiler, we should be able to have typing information when attempting to import various Dijit modules.
+
+{% section %}
+
+## Configuring Dojo 1
+
+{% task 'Create loader configuration.' %}
+
+{% aside 'Information' %}
+The Dojo 1 loader configuration can be complex.  There is this Dojo 1 tutorial <em>[Configuring Dojo with dojoConfig](https://dojotoolkit.org/documentation/tutorials/1.10/dojo_config/index.html)</em> that explains how to configure Dojo 1.
+{% endaside %}
+
+In the previous step, we need to include a configuration for our Dojo 1 loader.  For this tutorial, we only need to ensure that the Dojo 1 loader works in an asynchronous fashion, because the Dojo 1 loader will default to synchronous operation, which will perform slower and cause issues with our application.  The file could be named anything, but in this tutorial, we are naming it `/src/dojo1.cfg.js`.
+
+{% instruction 'Create a `/src/dojo1.cfg.js` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dojo1.cfg.js' %}
+
+{% task 'Ensure the application builds.' %}
+
+At this point, you should be able to build your application.
+
+{% instruction 'Run `dojo build` in the project root.' %}
+
+{% aside 'Reminder' %}
+You can now use `dojo build -w` to continue to develop at this point.
+{% endaside %}
+
+You should see a rather long output indicating lots of files have been output.
+
+{% section %}
+
+## Dealing with Dijits
+
+In order for Dijits to be integrated into a Dojo 2 application, they need to _wrapped_ so that they can be managed by the Dojo 2 widgeting system.  As discussed in previous steps, there is a function named `DijitWrapper` which will _convert_ a Dojo 1 Dijit constructor function into something that behaves like a Dojo 2 widget class.  When used with the Dojo 1 typings, you will also get code intellisense when configuring your Dijits.
+
+The `DijitWrapper` function takes two arguments.  The first is the Dojo 1 `declare` class.  The second is an optional string, which represents that tag name that should be used when the Dojo 2 widgeting system needs to create a _DOM stub_ to attach the Dijit instance to.  It defaults to `div` if not supplied.
+
+{% task 'Wrap Dijits.' %}
+
+{% aside 'Information' %}
+All Dojo 1 modules are AMD modules without default exports.  Because of this, you have to import them using the `import * as Dijit from 'module/id';` syntax.
```

Comment:
Because of this, you have to import them using the `import * as Dijit from 'module/id';` syntax. -> As such, you need to import Dojo 1 modules using the `import * as Dijit from 'module/id';` syntax.